istio.VERSION

```diff
@@ -0,0 +1,6 @@
+# Do not edit that file manually instead use
+# scripts/update_version.sh -h
+INIT = "docker.io/istio/init:2017-03-22-21.21.48"
+MIXER = "docker.io/istio/mixer:2017-03-22-19.36.02"
+RUNTIME = "docker.io/istio/runtime:2017-03-22-21.21.48"
```

Comment:
actually I prefer keeping it as runtime until the docker image name is updated to manager, because that's even more confusing I think.